site/content/en/docs/commands/addons.md

```diff
@@ -128,6 +128,7 @@ minikube addons enable dashboard
 ### Options
 
 ```
+      --force               If true, will force gcp-auth addon to be enabled on GCE.
```

Comment:
this should be more generalized help text for all addons.
consider changing this to
```  Force minikube addons to perform possibly dangerous or not recommended operations at user's discretion```

Reply:
even though this is currently used by gcp Auth this is a flag for addons 
there could be a comment and once someone else adds a force check for addons, they will forget to update the help text.

Reply:
Fixed.